packages/react-percy-storybook/src/getStories.js

```diff
@@ -13,17 +32,19 @@ function getStoriesFromDom(previewJavascriptCode) {
                 resolve(window.__storybook_stories__);
             }
         };
-        // parse preview bundle js code and retrieve window object
+        if (options.debug) {
+            jsDomConfig.virtualConsole = jsdom.createVirtualConsole().sendTo(console);
+        }
         jsdom.env(jsDomConfig);
     });
 }
 
-export default async function getStories(assets) {
+export default async function getStories(assets, options) {
```

Comment:
If you do a `let options = options || {};` inside the function, you could drop the second arg to `getStories(assets, {})` above in `getStories-tests.js` and elsewhere.